node/core/dispute-coordinator/src/real/mod.rs

```diff
@@ -582,12 +582,12 @@ async fn handle_incoming(
 			.await?;
 		},
 		DisputeCoordinatorMessage::DetermineUndisputedChain {
-			base_number,
+			base: (base_number, base_hash),
```

Comment:
nit: split this into `base_number` and `base_hash` instead on the messages already, `base` is never used as tuple.

Reply:
They really belong together, so I liked the grouping to highlight that relationship.